src/components/sign-in/sign-in.jsx

```diff
@@ -0,0 +1,116 @@
+import React, {PureComponent, createRef} from 'react';
+import {connect} from 'react-redux';
+import PropTypes from 'prop-types';
+
+import {getErrorAuthorizationStatus} from '../../reducer/user/selectors';
+
+class SignIn extends PureComponent {
+  constructor(props) {
+    super(props);
+
+    this.loginRef = createRef();
+    this.passwordRef = createRef();
+
+    this._handleSubmit = this._handleSubmit.bind(this);
+  }
+
+  _renderErrorMessage() {
+    const {isErrorAuth} = this.props;
+
+    if (isErrorAuth) {
+      return (
+        <div className="sign-in__message">
+          <p>We can’t recognize this email <br/> and password combination. Please try again.</p>
+        </div>
+      );
+    }
+
+    return ``;
```

Comment:
```suggestion
    return null;
```

а еще лучше тернарный оператор